FirstPersonScience.vcxproj

```diff
@@ -40,13 +40,13 @@
     <CharacterSet>MultiByte</CharacterSet>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
-    <ConfigurationType>Application</ConfigurationType>
+    <ConfigurationType>StaticLibrary</ConfigurationType>
```

Comment:
In the spirit of keeping older workflows working I suggest we rename this project to `FPSciLib` or something equivalent to keep the `FirstPersonScience` as an application and introduce a new project for the library.

Reply:
done